src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -25,6 +25,11 @@ public abstract class AbstractAcquireTokenParameterBuilder<T>
     {
         internal IServiceBundle ServiceBundle { get; }
 
+        /// <summary>
+        /// Default constructor for AbstractAcquireTokenParameterBuilder
+        /// </summary>
```

Comment:
should add [Obsolete("Please use the derived classes to construct this object.")]

Reply:
@jmprieur thoughts?

Reply:
we don't need obsolete anymore.

---
In reply to: [480299373](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2022#discussion_r480299373) [](ancestors = 480299373)

Reply:
Ohh we dont?

Reply:
i don't think so. that causes the same issue. @jmprieur to make a final decision though.

Reply:
In this case, we don't need it as there are other constructors.
the reason why we read this constructor is to keep the binary compatibility.